frontend/src/components/pages/CreatePasswordPage.tsx

```diff
@@ -0,0 +1,202 @@
+                    textColor="var(--gray-600, #4A5568)"
+                    lineHeight="8px"
+                  >
+                    Email Address:
+                  </FormLabel>
+                  <Input
+                    textColor="var(--gray-400, #A0AEC0)"
+                    fontSize="14px"
+                    height="2.4rem"
+                    placeholder="admin@humanesociety.org"
+                    isDisabled
+                    bg="var(--gray-200, #E2E8F0)"
+                  />
+                </Box>
+                <Box fontSize="12px">
+                  <FormLabel
+                    textColor="var(--gray-600, #4A5568)"
+                    fontSize="14px"
+                    lineHeight="8px"
+                  >
+                    Create Password:
+                  </FormLabel>
+                  <FormControl isInvalid={!!errorMessage}>
+                    <ResponsivePasswordInput
+                      value={password}
+                      onChange={handlePasswordChange}
+                    />
+                  </FormControl>
+                </Box>
+                <Box fontSize="12px">
+                  <FormLabel
+                    textColor="var(--gray-600, #4A5568)"
+                    fontSize="14px"
+                    lineHeight="8px"
+                  >
+                    Confirm Password:
+                  </FormLabel>
+                  <FormControl isInvalid={!!errorMessage}>
+                    <ResponsivePasswordInput
+                      value={confirmPassword}
+                      onChange={handleConfirmPasswordChange}
+                    />
+                  </FormControl>
+                </Box>
+              </Stack>
+              <Box>
+                <Button
+                  type="submit"
+                  fontSize="14px"
+                  onClick={handleCreateAccount}
+                  color="white"
+                  h="2.4rem"
+                  width="100%"
+                  bg="var(--blue-700, #2C5282)"
+                >
+                  Create Account
+                </Button>
+                {errorMessage && (
+                  <Box textAlign="center">
+                    <Text color="red.500" fontSize="14px" lineHeight="1" mb="0" mt="1rem">
+                      {errorMessage}
+                    </Text>
+                  </Box>
+                )}
+              </Box>
+            </Stack>
+          </ResponsiveAuthContainer>
+        </Flex>
+      </Center>
+      {showModal && (
+        <ResponsiveModalWindow>
+          <Text
+            color="#2C5282"
+            textAlign="center"
+            textStyle={{ base: "h2", md: "h1" }}
+          >
+            Success!
+          </Text>
+          <Text
+            textStyle={{ base: "bodyMobile", md: "body" }}
+            textAlign="center"
+          >
+            Welcome to the Oakville & Milton Humane Society
+          </Text>
+          <Button
```

Comment:
This will link to the main page right? Please make a comment here for future reference. use a TODO: for easy searching